templates/config/config.sh.aws

```diff
@@ -0,0 +1,49 @@
+# Amazon Web Services
+# Kubernetes cluster creation definitions, generated by ./configure.
+# It's OK to modify this file as necessary to customize your deployment,
+# but note it may be overwritten if ./configure is run again.
+
+# Cloud hosting provider
+PROVIDER=aws				# Cloud hosting provider
+USERNAME=                   # The username associated with the current AWS profile (REQUIRED)
+
+# Kubernetes cluster definitions
+CLUSTER_NAME=genesis-test			# Kubernetes cluster name
+K8S_VERSION=1.14			# Kubernetes engine version
```

Comment:
Is this the latest on AWS?

Reply:
Yes

Reply:
https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html